src/pickley/bstrap.py

```diff
@@ -63,28 +62,16 @@ def find_base(base):
         if base:
             print("Found existing %s" % short(base))
             return os.path.dirname(base)
+
         base = DEFAULT_BASE
 
-    path_dirs = os.environ.get("PATH", "").split(os.pathsep)
     candidates = base.split(os.pathsep)
     for c in candidates:
         c = os.path.expanduser(c)
-        if c and os.path.isdir(c) and is_writable(c) and c in path_dirs:
+        if c and os.path.isdir(c) and is_writable(c):
             return c
 
-    msg = "Make sure %s is writeable and in your PATH variable." % candidates[0]
-    abort(msg)
-
-
-def find_python3():
-    if sys.version_info[0] == 3 and not RUNNING_FROM_VENV:
-        # We're not running from a venv
-        return sys.executable
-
-    if is_executable("/usr/bin/python3"):
-        return "/usr/bin/python3"
-
-    return which("python3")
+    abort("Make sure %s is writeable." % candidates[0])
```

Comment:
Yes, up until this merge `bstrap.py` could have been ran by py2, and it would find py3 (or complain) and install pickley with py3.
After this change, the user must invoke `bstrap.py` with the python they wish pickley to install itself with

The modulo operator formatting is minor, will leave it for now, but yes f-strings could be used from now on everywhere